packages/overlay/test/overlay.test.js

```diff
@@ -11,7 +11,6 @@ import {
 } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import '@vaadin/polymer-legacy-adapter/template-renderer.js';
-import '@vaadin/text-field/vaadin-text-field.js';
```

Comment:
This import was unused, apparently because of some previous changes to the tests.